lib/libc/sys/Makefile.inc

```diff
@@ -6,12 +6,7 @@
 
 # Include the generated makefile containing the *complete* list
 # of syscall names in MIASM.
-.if ${MACHINE_ABI:Mpurecap}
```

Comment:
This is the only non-nop change.  A purecap libc will contain a few more compat syscall stubs than it used to.  I don't think it hurts though (and said syscalls do exist in the native ABI).

Reply:
This is fine now that compat things are working.  Someday I'd like us to have a framework to not do compat for things that never existed for an architecture/ABI, but not today.

Reply:
Yes, agreed.  I thought we already had something to not include pipe, etc. for aarch64() and riscv?

BTW, there is one thing I didn't change later in this file where we omit break.c for purecap.  I think that's probably fine to stay as true, though it probably should really be under the same framework rather than conditional on purecap.